samples/sgx-ra/CMakeLists.txt

```diff
@@ -0,0 +1,80 @@
+# Copyright (C) 2019 Intel Corporation.  All rights reserved.
```

Comment:
Could you please write a `README.md` under this folder to introduce the sample?

Another issue is that do I need to install other packages? I got compilation error while building this sample:
```
samples/sgx-ra/build/_deps/librats-src/tee/sgx/untrust/sgx_ecdsa_ocall.c:14:10: fatal error: sgx_dcap_quoteverify.h: No such file or directory
   14 | #include <sgx_dcap_quoteverify.h>
```

Reply:
Done